CONTRIBUTING.md

```diff
@@ -9,47 +9,46 @@ We welcome contributions from the community. Here are some guidelines.
-  guidelines.
-
-# Deviations from Google C++ Style guidelines
-
-* Exceptions are allowed and encouraged where appropriate.
-* References are always preferred over pointers when the reference cannot be null. This
-  includes both const and non-const references.
-* Function names using camel case starting with a lower case letter (e.g., "doFoo()").
-* Struct/Class member variables have a '\_' postfix (e.g., "int foo\_;").
-* 100 columns is the line limit.
-* Global static non-pod variables are allowed because Envoy correctly joins all threads on exit.
-  However, care is needed during init and static accessor functions may be required.
-* OOM events (both memory and FDs) are considered fatal crashing errors.
-* Use your GitHub name in TODO comments, e.g. `TODO(foobar): blah`.
-* Smart pointers are type aliased:
-  * `typedef std::unique_ptr<Foo> FooPtr;`
-  * `typedef std::shared_ptr<Bar> BarSharedPtr;`
-  * `typedef std::shared_ptr<const Blah> BlahConstSharedPtr;`
-  * Regular pointers (e.g. `int* foo`) should not be type aliased.
-* API-level comments should follow normal Doxygen conventions. Use `@param` to describe
-  parameters, `@return <return-type>` for return values.
-* There are probably a few other things missing from this list. We will add them as they
-  are brought to our attention.
+* See [STYLE.md](STYLE.md)
+
+# Breaking configuration change policy
+
+* As of the 1.3.0 release, the Envoy configuration is locked and we will not make breaking changes
+  between releases.
+* We reserve the right to deprecate configuration, and at the beginning of the following release
+  cycle remove the deprecated configuration.
+* All deprecations/breaking changes will be clearly listed in the release notes.
+
+# Release cadence
+
+* Currently we are targeting approximately quarterly official releases. We may change this based
+  on customer demand.
+* In general, master is assumed to be release candidate quality at all times for documented
+  features. For undocumented or clearly under development features, use caution or ask about
+  current status when running master. Lyft runs master in production, typically deploying every
+  few days.
+
+# PR review policy
+
+* Typically we try to turn around reviews within one business day.
+* See [OWNERS.md](OWNERS.md) for ther current list of committers.
+* If possible, a senior committer should review every PR.
+* Anyone is welcome to review any PR that they want, whether they are a committer or not.
+* Committers **must** verify that the PR author has signed the CLA. Currently we do not have a
+  bot that verifies CLA compliance, and only Lyft employees can check the CLA tool to see if
+  someone has signed. Non-Lyft committers need to ask a Lyft employee to check unless they are sure
+  the author has signed.
 
 # Submitting a PR
 
 * Fork the repo and create your PR.
-* Tests will automatically run for you. 
+* Tests will automatically run for you.
```

Comment:
"No commit will be merged for any reason unless all tests pass." Probably goes without saying with the current contributors, but as we scale it should be a hard rule.